war/pom.xml

```diff
@@ -736,7 +740,7 @@ THE SOFTWARE.
                   <goal>gulp</goal>
                 </goals>
                 <configuration>
-                  <arguments>lint bundle</arguments>
+                  <arguments>jshint bundle</arguments>
```

Comment:
the latest version of `js-builder` [defines](https://github.com/jenkinsci/js-builder/blob/master/index.js#L139) `lint` task instead of `jshint`.